app/helpers/applications_helper.rb

```diff
@@ -68,6 +70,10 @@ def page_title(application)
     "#{@application.address} | #{@application.date_scraped.to_date.to_formatted_s(:rfc822)}"
   end
 
+  def applications_json_path(authority)
```

Comment:
I think it would be good to name this something more descriptive, even if it's more verbose, like `authority_applications_json_path_for_current_user(authority)` so it's super clear what you're gonna get in the view.